sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2224,6 +2305,67 @@ public partial class SenderAuthorization
         public string Role { get { throw null; } }
         public string Scope { get { throw null; } }
     }
+    public partial class SubscriptionMonitorMetric
+    {
+        internal SubscriptionMonitorMetric() { }
+        public string DisplayDescription { get { throw null; } }
+        public string ErrorCode { get { throw null; } }
+        public string ErrorMessage { get { throw null; } }
+        public string Id { get { throw null; } }
+        public Azure.ResourceManager.Monitor.Models.MonitorLocalizableString Name { get { throw null; } }
+        public string SubscriptionScopeMetricType { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.Monitor.Models.MonitorTimeSeriesElement> Timeseries { get { throw null; } }
+        public Azure.ResourceManager.Monitor.Models.MonitorMetricUnit Unit { get { throw null; } }
+    }
+    public partial class SubscriptionResourceGetMonitorMetricsOptions
+    {
+        public SubscriptionResourceGetMonitorMetricsOptions(string region) { }
+        public string Aggregation { get { throw null; } set { } }
+        public bool? AutoAdjustTimegrain { get { throw null; } set { } }
+        public string Filter { get { throw null; } set { } }
+        public System.TimeSpan? Interval { get { throw null; } set { } }
+        public string Metricnames { get { throw null; } set { } }
+        public string Metricnamespace { get { throw null; } set { } }
+        public string Orderby { get { throw null; } set { } }
+        public string Region { get { throw null; } }
+        public Azure.ResourceManager.Monitor.Models.MonitorMetricResultType? ResultType { get { throw null; } set { } }
+        public string Timespan { get { throw null; } set { } }
+        public int? Top { get { throw null; } set { } }
+        public bool? ValidateDimensions { get { throw null; } set { } }
+    }
+    public partial class SubscriptionResourceGetMonitorMetricsWithPostContent
```

Comment:
Weird name, any idea of a better name?

Reply:
these names are based on pattern: "{containing resource name}{method name}Content"
Because they already have a `GetMonitorMetrics` method (with GET), this method has the same fucntionality but using `Post`, I could only come up with this name.
Do you have a suggestion?